web/templates/eula.html.tmpl

```diff
@@ -0,0 +1,37 @@
+{{ define "content" }}
+    <div class="mb-4">
+        <div class="row">
+            <div class="col-sm-8">
+                <h1 class='display-4 lead'>{{ .Title }}</h1>
+                <p class='subheadline'>An open cloud-native web console improving the
+                  life of SAP Applications administrators</p>
+                <hr/>
+                <p>Trento is an open source (GPL-3.0) component that is supported as part of SUSE Linux Enterprise Server for SAP applications and therefore falls under the standard SUSE Terms and Conditions that can be reviewed at <a href="https://www.suse.com/products/terms_and_conditions.pdf">https://www.suse.com/products/terms_and_conditions.pdf</a>, SUSE EULA is available at <a href="https://www.suse.com/licensing/eula/#server">https://www.suse.com/licensing/eula/#server</a>.</p>
+
+                <p>By using Trento via a registered SUSE Linux Enterprise Server for SAP you agree to the Terms and Conditions also for Trento.</p>
+
+
+                <p>Trento collects the following KPIs on your environment and transmits it to SUSE:<p>
+                <ul>
+                  <li>SUSE Customer Center ID</li>
```

Comment:
We don't store the `SUSE customer center ID` by now. Do we want to store this. It looks against privacy hehe

Reply:
People high above me gave me this text, I pasted inside barely reading it. Maybe we can address this in a second moment, looks quite tricky to me :smile:

Reply:
@stefanotorresi @lee-martin @diegoakechi What about this?

Reply:
@arbulu89 @dottorblaster 

Below is the license agreement requested by PM and approved by legal, I'll forward internally as well.

Regarding SCC ID, isn't that actually a host detail? I mean each host could well have different SCC IDs?

"
License Agreement

Trento is an open source (GPL-3.0) component that is supported as part of SUSE Linux Enterprise Server for SAP applications and therefore falls under the standard SUSE Terms and Conditions that can be reviewed at https://www.suse.com/products/terms_and_conditions.pdf, SUSE EULA is available at https://www.suse.com/licensing/eula/#server .

By using Trento via a registered SUSE Linux Enterprise Server for SAP you agree to the Terms and Conditions also for Trento.

Trento collects the following KPIs on your environment and transmits it to SUSE:

-          SUSE Customer Center ID
-          Number of registered hosts in Trento
-          For each of those hosts:
o   SLES version
o   Number of sockets and total count of cores per host
o   Available memory per host
o   Hosting platform (Cloud service provider, Hypervisor or bare Metal)

Trento neither collects, processes or sends any personal data to SUSE.

By using Trento and updates available through SUSE channels you agree to these terms. In case you disagree, please switch to the OpenSource version of Trento.
"

Reply:
@lee-martin No, the SCC ID is the ID you are given from SUSE to connect to SUSE customer center. It is of course private. With that ID, we could use their subscriptions.
I think that legal just re-phrased the content we just gave to them, and we might've given them this bullet list.
That's why we don't store it now. We could very well anonymize this data, but in this case, we wouldn't be storing the SCC ID itself.
At least, if it was myself, I wouldn't accept this EULA hehe

Reply:
@arbulu89 I don't think anyone is referring to the SCC access token itself, but moreover SUSE customer ID. Let me check this, perhaps the expression "SCC ID" needs clarifying or rewording.

What I meant about the ID being a "host detail" is that we cannot assume that all hosts in Trento use the same SCC/customer ID, therefore we should surely consider the ID per host and not only once globally?

Reply:
We don't store any SUSE customer ID, as I don't even know if this exist as such.
As an identifier, we auto-generate a random UUID based on the `/etc/machine-id` content.

Reply:
I think we can remove the "SUSE Customer Center ID" line entirely.

Reply:
@arbulu89 let's open a follow up to implement the SCC ID collection when available. We don't need to scrub it.

Reply:
@stefanotorresi I don't think this value is collectable from within the machines. And again!!! Most of our users are PAYG users. They are not SUSE customers directly. This ID might not even exist